test/Microsoft.Azure.WebJobs.Extensions.ServiceBus.Tests/Bindings/ServiceBusTriggerBindingTests.cs

```diff
@@ -1,101 +0,0 @@
-﻿// Copyright (c) .NET Foundation. All rights reserved.
-// Licensed under the MIT License. See License.txt in the project root for license information.
-
-using System;
-using System.Collections.Generic;
-using System.IO;
-using System.Text;
-using Microsoft.Azure.WebJobs.ServiceBus.Triggers;
-using Microsoft.Azure.ServiceBus;
-using Xunit;
-using Microsoft.Azure.ServiceBus.Core;
-using Moq;
-
-namespace Microsoft.Azure.WebJobs.ServiceBus.UnitTests.Bindings
-{
-    public class ServiceBusTriggerBindingTests
-    {
-        [Fact]
-        public void CreateBindingDataContract_ReturnsExpectedValue()
-        {
-            IReadOnlyDictionary<string, Type> argumentContract = null;
```

Comment:
I don't see where you retained these tests?

Reply:
there are no such thing as thing `ServiceBusTriggerBinding`, Now it's `ServiceBusTriggerBindingStrategy`